lib/msf/core/exploit/http/wordpress/version.rb

```diff
@@ -96,9 +97,8 @@ def check_version_from_custom_file(uripath, regex, fixed_version = nil, vuln_int
       'method' => 'GET'
     )
 
-    # file not found
     unless res && res.code == 200
-      return Msf::Exploit::CheckCode::Unknown
+      return Msf::Exploit::CheckCode::Unknown("File not found")
```

Comment:
Probably should check the response code is 404 before you can know for sure the issue was "File not Found"

Reply:
Absolutely, I changed the message to be less specific and keep the original the logic here (857677f39d).